maxim/apis/maxim_apis.py

```
@@ -1576,3 +1741,89 @@ def upload_to_signed_url(self, url: str, data: bytes, mime_type: str) -> bool:
         if last_exception:
             raise Exception(f"File upload failed: {last_exception}")
         raise Exception("File upload failed for unknown reasons")
+
+    def create_dataset_entries(self, dataset_id: str, entries: List[Dict[str, Any]]) -> dict[str, Any]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)